packages/featureserver/src/helpers/normalize-request-params.js

```diff
@@ -0,0 +1,46 @@
+const _ = require('lodash');
+const { MAX_RECORD_COUNT } = require('../constants');
+
+function normalizeRequestParameters (
+  query,
+  body,
+  maxRecordCount = MAX_RECORD_COUNT,
+) {
+  const definedQueryParams = _.chain(query)
+    .pickBy(isNotEmptyString)
+    .mapValues(coerceStrings)
+    .value();
+
+  const { resultRecordCount, ...params } = { ...definedQueryParams, ...body };
```

Comment:
Is there concern that body params will overwrite query params?

Reply:
That's okay and consistent with Geoservice specification I believe.

Reply:
👍